packages/nocodb/src/db/BaseModelSqlv2/relation-data-fetcher.ts

```
@@ -0,0 +1,1557 @@
+          await Model.getBaseModelSQL(baseModel.context, {
+            id: childTable.id,
+            dbDriver: baseModel.dbDriver,
+          })
+        ).getProto();
+
+        return children.map((c) => {
+          c.__proto__ = proto;
+          return c;
+        });
+      } catch (e) {
+        throw e;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)